mixer/template/BUILD

```diff
@@ -12,8 +12,10 @@ mixer_supported_template_library(
         "//mixer/template/reportnothing:go_default_library_proto.descriptor_set": "istio.io/istio/mixer/template/reportnothing",
         "//mixer/template/checknothing:go_default_library_proto.descriptor_set": "istio.io/istio/mixer/template/checknothing",
         "//mixer/template/apikey:go_default_library_proto.descriptor_set": "istio.io/istio/mixer/template/apikey",
+        "//mixer/adapter/svcctrl/template/svcctrlreport:go_default_library_proto.descriptor_set": "istio.io/istio/mixer/adapter/svcctrl/template/svcctrlreport",
```

Comment:
nit: add this to the top of the list to keep alphabetical sorting.

Reply:
Strange, buildifier didn't do the fix. done.